webdriver/client.py

```diff
@@ -385,12 +377,13 @@ def send_command(self, method, uri, body=None):
 
         url = urlparse.urljoin("session/%s/" % self.session_id, uri)
         response = self.send_raw_command(method, url, body)
+        value = response.body["value"]
 
-        rv = response.body["value"]
-        if not rv:
-            rv = None
+        if response.status != 200:
```

Comment:
Thanks for a good summary @jugglinmike. I think that makes sense. I have pushed https://github.com/w3c/wdclient/pull/11/commits/d50a2cbe8f879aa581e1f860ade3afc0f3c59081 as a follow-up. Is this in the direction you imagined?